ports/raspberrypi/boards/mtm_computer/mpconfigboard.mk

```diff
@@ -0,0 +1,13 @@
+USB_VID = 0x239A
+USB_PID = 0x10C1
+USB_PRODUCT = "Workshop Computer"
+USB_MANUFACTURER = "Music Thing Modular"
+
+CHIP_VARIANT = RP2040
+CHIP_FAMILY = rp2
+
+EXTERNAL_FLASH_DEVICES = "W25Q16JVxQ"
+
+CIRCUITPY_AUDIODELAYS = 1
```

Comment:
Do you just want the delays? `CIRCUITPY_AUDIOEFFECTS = 1` will give you filters (and future ones) as well.

Reply:
You could get rid of the AUDIODELAYS line (effects should contain filters and delays), but doesn't do any harm having it there.